src/components/experiences/social-login/index.js

```diff
@@ -27,13 +32,7 @@ function SocialLogin(props) {
                 <GoogleIcon className="rainbow-auth-firebase-social-login_social-icon" />
                 Login with Google
             </Button>
-            <Button
-                variant="neutral"
-                className="rainbow-auth-firebase-social-login_button rainbow-auth-firebase-social-login_amazon-button"
-                onClick={() => {}}>
-                <AmazonIcon className="rainbow-auth-firebase-social-login_social-icon" />
-                Login with Amazon
-            </Button>
+            <Select className="rainbow-auth-firebase-social-language" options={languages} onChange={value => changeLocale(value)} />
```

Comment:
here you need to use:
```onChange={event => changeLocale(event.target.value)}```
also you need to pass it trough props destructuring since right now you are calling the action directly not the one that is passed from props, so it is not binded with dispatch.
and what value is selected here?
you must to get the locale form the i18n store and set it as value in select.
we need this working!!!